components/LoadingContainer/LoadingContainer.spec.jsx

```diff
@@ -0,0 +1,35 @@
+// Mount import
+import { mount } from '@cypress/react';
+
+// Component import
+import LoadingContainer from './LoadingContainer';
+
+/**
+ * Loading container component tests.
+ */
+describe('Loading Container', () => {
+  it('No loading indicator by default', () => {
+    mount(
```

Comment:
Can this be moved to some function so that we don't repeat the same code in each test?

Reply:
Example of extracting the `mount` call from each test can be seen in the `UserInformation.spec.jsx`. We extracted it there because the `mount` call is the same in each test. Meanwhile, here, I don't see the need for such thing as the mounting component is different. (Second test has `isLoading` component prop set to `true`)

Reply:
Sorry, I completely overlooked that prop.